net/sched/p4tc/p4tc_tbl_api.c

```diff
@@ -1639,12 +1628,10 @@ static int tcf_table_entry_dump(struct sk_buff *skb, struct nlattr *arg,
 		return ret;
 	}
 
-	rcu_read_lock();
 	if (!ctx->iter) {
 		ctx->iter = kzalloc(sizeof(*ctx->iter), GFP_ATOMIC);
```

Comment:
Could you also change `GFP_ATOMIC` to `GFP_KERNEL`

Reply:
Done